optaplanner-website-docs/supplemental-ui/partials/header-content.hbs

```diff
@@ -0,0 +1,30 @@
+<header class="header">
+  <nav class="navbar">
+    <div class="navbar-brand">
+      <div class="navbar-item">
+        <a href="{{{uiRootPath}}}/../../" style="width:20%">
+          <img src="{{{uiRootPath}}}/img/optaPlannerIcon.svg" alt="OptaPlanner logo" style="width:50%">
+        </a>
+        <a class="navbar-item" href="{{{or site.url (or siteRootUrl siteRootPath)}}}">{{site.title}}</a>
+      </div>
+      <div class="navbar-item hide-for-print">
+        <input id="search-input" type="text" placeholder="Search docs">
+      </div>
+      <button class="navbar-burger" data-target="topbar-nav">
+        <span></span>
+        <span></span>
+        <span></span>
+      </button>
+    </div>
+    <div id="topbar-nav" class="navbar-menu">
+      <div class="navbar-end">
+        <a class="navbar-item" href="https://docs.optaplanner.org">Archive</a>
+      </div>
+    </div>
+    <div id="topbar-nav" class="navbar-menu">
+          <div class="navbar-end">
+            <a class="navbar-item" href="TODO">Product documentation</a>
```

Comment:
There's a TODO href here.

Reply:
Good catch!

@emmurphy1 could you please advise the correct product documentation link?